{
  try {
    _persistence.countByE1_T(RandomTestUtil.nextLong(),RandomTestUtil.nextInt());
    _persistence.countByE1_T(0L,0);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
